app/src/main/java/com/peacecorps/pcsa/reporting/HomeScreen.java

```diff
@@ -51,16 +52,20 @@ public void onClick(View v) {
         reportingMoreResources.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                Intent types = new Intent(HomeScreen.this, Resources.class);
-                startActivity(types);
+                //Commented out because screen is blank.
+                Toast.makeText(HomeScreen.this, "This functionality is currently unavailable.", Toast.LENGTH_SHORT).show();
+                //Intent types = new Intent(HomeScreen.this, Resources.class);
+                //startActivity(types);
```

Comment:
This is a screen containing some static data. Content for this screen is pending from PCSA. Do not remove.